node/src/peer_tracker.rs

```diff
@@ -157,10 +215,40 @@ impl PeerTracker {
             // collect instead of returning an iter to not block the dashmap
             .collect()
     }
+
+    pub fn trusted_n_peers(&self, limit: usize) -> Vec<PeerId> {
+        self.peers
+            .iter()
+            .filter(|pair| pair.value().is_connected() && pair.value().trusted)
+            .take(limit)
+            .map(|pair| pair.key().to_owned())
+            // collect instead of returning an iter to not block the dashmap
+            .collect()
+    }
 }
 
 impl Default for PeerTracker {
     fn default() -> Self {
         PeerTracker::new()
     }
 }
+
+fn increase_connected_peers(info_tx: &watch::Sender<PeerTrackerInfo>, trusted: bool) {
```

Comment:
naming nit: maybe `increment/decrement`, drives home the small, definite (1) amount of change. 
Or maybe change entirely to `notify_peer_connected/disconnected` to focus on the intent rather than implementation.

Reply:
Done. I chose `increment/decrement`.